packages/tocco-ui/src/Layout/StyledLayoutContainer.js

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components'
+import {theme} from 'styled-system'
+
+const declareStylesIfNestedCorrectly = props => {
```

Comment:
const declareStylesIfNestedCorrectly = props => 
  props.isNestedCorrectly ? 
    `
      display: flex;
      flex-wrap: wrap;
      margin: 0 -${theme('space.4')(props)};
      margin: 0 -15px;
       &:last-child {
        margin-bottom: 0;
      }
    `
  :
    `
      display: flex;
      flex-wrap: wrap;
      width: 100%;
    `
  }
}


would be shorter, but thats just a detail

Reply:
Personally, I prefer readable over compact css. Build process shall remove unnecessary whitespace, which it does most likely already.

Reply:
dont get me wrong. its not about the css (all on one line happend when i copy&paste and should not be done).
Its about replaceing

const xy = props => {
if (x) {return y} else {return z}
}

with const xy = props =>
  x? y : z